server/api/routes/validators/users.py

```diff
@@ -0,0 +1,45 @@
+from email_validator import validate_email, EmailNotValidError
+
+def valid_email(email):
+    """Checks if an email is valid
+
+    Args:
+        email (str): user's email address
+
+    Returns:
+        (True, email): normalized email if it is valid
```

Comment:
What does normalized mean? Does it lowercase it all or smth?

Reply:
Surprisingly, since Unicode is allowed for emails two different strings can mean the same thing, so normalizing them keeps things consistent.

```py
valid = validate_email("me@Ｄｏｍａｉｎ.com")
email = valid.ascii_email
print(email)
# prints: me@domain.com
```

Reply:
Oh damn that's cool